package.json

```diff
@@ -46,37 +46,162 @@
     ],
     "permissionSets": [
       {
-        "permissionName": "module.finance.enabled",
-        "displayName": "UI: Finance module is enabled",
-        "visible": true
+        "permissionName": "ui-finance.third-party-services",
```

Comment:
this is anti pattern per permissions wiki https://wiki.folio.org/display/DD/Permission+Set+Guidelines#PermissionSetGuidelines-UIModules